test/dpt_tests/dpt_string_test.py

```diff
@@ -89,3 +89,11 @@ def test_from_knx_wrong_parameter_length(self, raw):
         """Test parsing of KNX string with wrong elements length."""
         with pytest.raises(ConversionError):
             DPTString.from_knx(raw)
+
+    
+    def test_no_unit_of_measurement(self):
+        """Test for no unit set for DPT 16"""
+        assert DPTString.unit == None
```

Comment:
```suggestion
        assert DPTString.unit is None
```
Use identity comparison instead of equality. You can remove the not-equality checks for `""` then.

Reply:
Change is done and comitted.